concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+    Then <root-type>(<subtype-name>) get owns(reference) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name>) get owns(work-email) get annotations contain: @<annotation>
+    When put attribute type: license
+    When attribute(license) set value-type: string
+    When attribute(license) set supertype: reference
+    When put attribute type: points
+    When attribute(points) set value-type: double
+    When attribute(points) set supertype: rating
+    When <root-type>(<subtype-name-2>) set supertype: <subtype-name>
+    When <root-type>(<subtype-name-2>) set owns: license
+    When <root-type>(<subtype-name-2>) get owns(license) set override: reference
+    When <root-type>(<subtype-name-2>) set owns: points
+    When <root-type>(<subtype-name-2>) get owns(points) set override: rating
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then <root-type>(<subtype-name>) get owns contain:
+      | username   |
+      | reference  |
+      | work-email |
+      | age        |
+      | rating     |
+      | nick-name  |
+    Then <root-type>(<subtype-name>) get owns do not contain:
+      | email |
+      | name  |
+    Then <root-type>(<subtype-name>) get declared owns contain:
+      | reference  |
+      | work-email |
+      | rating     |
+      | nick-name  |
+    Then <root-type>(<subtype-name>) get declared owns do not contain:
+      | username |
+      | age      |
+      | email    |
+      | name     |
+    Then <root-type>(<subtype-name>) get owns(username) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name>) get owns(reference) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name>) get owns(work-email) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name-2>) get owns overridden(license) get label: reference
+    Then <root-type>(<subtype-name-2>) get owns overridden(points) get label: rating
+    Then <root-type>(<subtype-name-2>) get owns contain:
+      | username   |
+      | license    |
+      | work-email |
+      | age        |
+      | points     |
+      | nick-name  |
+    Then <root-type>(<subtype-name-2>) get owns do not contain:
+      | email     |
+      | reference |
+      | name      |
+      | rating    |
+    Then <root-type>(<subtype-name-2>) get declared owns contain:
+      | license |
+      | points  |
+    Then <root-type>(<subtype-name-2>) get declared owns do not contain:
+      | username   |
+      | work-email |
+      | age        |
+      | nick-name  |
+      | email      |
+      | reference  |
+      | name       |
+      | rating     |
+    Then <root-type>(<subtype-name-2>) get owns(username) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name-2>) get owns(license) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name-2>) get owns(work-email) get annotations contain: @<annotation>
+    Examples:
```

Comment:
Feel free to ask me to shorten all these Examples if you find their amount too extreme.

Reply:
All good!